packages/ui5-application-inquirer/src/prompts/prompts.ts

```diff
@@ -98,14 +98,19 @@ export function getQuestions(
  * @param capCdsInfo CDS UI5 plugin information
  * @returns The `enableTypeScript` prompt
  */
-function getEnableTypeScriptPrompt(capCdsInfo?: CdsUi5PluginInfo): UI5ApplicationQuestion {
+export function getEnableTypeScriptPrompt(capCdsInfo?: CdsUi5PluginInfo): UI5ApplicationQuestion {
     return {
-        when: (answers): boolean => {
+        when: (): boolean => {
             if (capCdsInfo) {
-                return capCdsInfo.isCdsUi5PluginEnabled || !!answers?.enableNPMWorkspaces;
+                return capCdsInfo.isCdsUi5PluginEnabled || (capCdsInfo.hasMinCdsVersion && !capCdsInfo.hasCdsUi5Plugin);
             }
             return true;
         },
+        additionalMessages: (val: boolean) => {
+            if (val && !capCdsInfo?.isWorkspaceEnabled) {
```

Comment:
This will be shown for all projects since `capCdsInfo` will not be defined. What is `isWorkspaceEnabled` ? It does not exist on the type: https://github.com/SAP/open-ux-tools/blob/7b73761fb5bd60edcf210467470a570f8429a1d7/packages/cap-config-writer/src/cap-config/types.ts#L3

Reply:
Thank you, I have added the change.